### PR TITLE
ci: Fix executor image build

### PR DESCRIPTION
**Pull request by eseliger**

### CODE REVIEW

File: enterprise/cmd/executor/cloudbuild/build.sh [hunk 1]
```diff
@@ -88,12 +92,6 @@ Environment=EXECUTOR_FIRECRACKER_IMAGE="${EXECUTOR_FIRECRACKER_IMAGE}"
 [Install]
 WantedBy=multi-user.target
 EOF
-
-  # Move binary into PATH
-  mv /tmp/executor /usr/local/bin
-
-  # Create configuration fiel and stub environment file
-  echo "${SERVICE_CONFIGURATION}" >>/etc/systemd/system/executor.service
```
Comment (tsenart) [inline enterprise/cmd/executor/cloudbuild/build.sh:96]: What was breaking with the previous approach?

**Review by tsenart — APPROVED** [review]

File: enterprise/cmd/executor/cloudbuild/build.sh (hunk 1 above)
Comment (eseliger) [reply enterprise/cmd/executor/cloudbuild/build.sh:96]: You cannot assign with << into a variable, and there are no spaces between `variable_name` and `=` allowed. I am no bash expert, this just gets it running again for now 😬

Comment (tsenart) [reply enterprise/cmd/executor/cloudbuild/build.sh:96]: Would shellcheck catch this?

Comment (eseliger) [reply enterprise/cmd/executor/cloudbuild/build.sh:96]: It somehow didn't 🤔

**Pull request merged**